extensions/amp-story/1.0/test/test-amp-story-grid-layer.js

```diff
@@ -127,6 +127,28 @@ describes.realWin('amp-story-grid-layer', {amp: true}, (env) => {
     ).to.equal(562);
   });
 
+  it('should use the scaling factor to set the size of the layer', async () => {
+    gridLayerEl.setAttribute('aspect-ratio', '1:2');
+    gridLayerEl.setAttribute('scaling-factor', '2');
```

Comment:
Can you change for a float since that's likely what publishers will use?
How about a non digits attribute? I know we'll add validation to enforce that but it should be simple to check

Reply:
Will change this test to float.

If it's not a float, the buildCallback will fail which is equal to triggering an error. That might be better than silently failing, since users will know why it's not working on the first place. WDYT?

Reply:
Commited change to check the scaling-factor parses a float, so now it will ignore the attribute if the value is not a float.